addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -247,9 +247,9 @@ export class AnalyticDistribution extends Component {
 
         if (searchLimit < records.length) {
```

Comment:
I guess the "View All" option should always be visible?

Reply:
Yes, this task is more than a string change :)

the idea is to make it easier for users to open a 'full view' to distinguish records that may be similarly named (and thus the list view allows them to better discriminate between records). So indeed the view all should be shown whenever the dropdown contains *something*

Hope this answers your question!